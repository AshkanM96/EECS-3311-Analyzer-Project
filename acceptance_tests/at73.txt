equals
boolean_constant(true)
boolean_constant(true)
type_check
evaluate
reset
equals
boolean_constant(true)
boolean_constant(false)
type_check
evaluate
reset
equals
boolean_constant(false)
boolean_constant(true)
type_check
evaluate
reset
equals
boolean_constant(false)
boolean_constant(false)
type_check
evaluate
reset
equals
integer_constant(-1)
integer_constant(-1)
type_check
evaluate
reset
equals
integer_constant(5)
integer_constant(7)
type_check
evaluate
