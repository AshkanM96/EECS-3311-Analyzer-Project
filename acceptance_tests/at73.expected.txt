  Expression currently specified: ?
  Report: Expression is initialized.
->equals
  Expression currently specified: (? = nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True = ?)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True = True)
  Report: OK.
->type_check
  Expression currently specified: (True = True)
  Report: (True = True) is type-correct.
->evaluate
  Expression currently specified: (True = True)
  Report: True
->reset
  Expression currently specified: ?
  Report: OK.
->equals
  Expression currently specified: (? = nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (True = ?)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (True = False)
  Report: OK.
->type_check
  Expression currently specified: (True = False)
  Report: (True = False) is type-correct.
->evaluate
  Expression currently specified: (True = False)
  Report: False
->reset
  Expression currently specified: ?
  Report: OK.
->equals
  Expression currently specified: (? = nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (False = ?)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (False = True)
  Report: OK.
->type_check
  Expression currently specified: (False = True)
  Report: (False = True) is type-correct.
->evaluate
  Expression currently specified: (False = True)
  Report: False
->reset
  Expression currently specified: ?
  Report: OK.
->equals
  Expression currently specified: (? = nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (False = ?)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (False = False)
  Report: OK.
->type_check
  Expression currently specified: (False = False)
  Report: (False = False) is type-correct.
->evaluate
  Expression currently specified: (False = False)
  Report: True
->reset
  Expression currently specified: ?
  Report: OK.
->equals
  Expression currently specified: (? = nil)
  Report: OK.
->integer_constant(-1)
  Expression currently specified: (-1 = ?)
  Report: OK.
->integer_constant(-1)
  Expression currently specified: (-1 = -1)
  Report: OK.
->type_check
  Expression currently specified: (-1 = -1)
  Report: (-1 = -1) is type-correct.
->evaluate
  Expression currently specified: (-1 = -1)
  Report: True
->reset
  Expression currently specified: ?
  Report: OK.
->equals
  Expression currently specified: (? = nil)
  Report: OK.
->integer_constant(5)
  Expression currently specified: (5 = ?)
  Report: OK.
->integer_constant(7)
  Expression currently specified: (5 = 7)
  Report: OK.
->type_check
  Expression currently specified: (5 = 7)
  Report: (5 = 7) is type-correct.
->evaluate
  Expression currently specified: (5 = 7)
  Report: False
